[2024-10-14T08:54:14.597+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-14T08:54:14.609+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: count_accidents.spark_count_number_of_accidents manual__2024-10-14T08:54:13.371924+00:00 [queued]>
[2024-10-14T08:54:14.613+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: count_accidents.spark_count_number_of_accidents manual__2024-10-14T08:54:13.371924+00:00 [queued]>
[2024-10-14T08:54:14.614+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-14T08:54:14.620+0000] {taskinstance.py:2888} INFO - Executing <Task(SparkSubmitOperator): spark_count_number_of_accidents> on 2024-10-14 08:54:13.371924+00:00
[2024-10-14T08:54:14.623+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1204) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-14T08:54:14.624+0000] {standard_task_runner.py:72} INFO - Started process 1206 to run task
[2024-10-14T08:54:14.625+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'count_accidents', 'spark_count_number_of_accidents', 'manual__2024-10-14T08:54:13.371924+00:00', '--job-id', '325', '--raw', '--subdir', 'DAGS_FOLDER/count_accidents_dag.py', '--cfg-path', '/tmp/tmp4ljunkv7']
[2024-10-14T08:54:14.625+0000] {standard_task_runner.py:105} INFO - Job 325: Subtask spark_count_number_of_accidents
[2024-10-14T08:54:14.657+0000] {task_command.py:467} INFO - Running <TaskInstance: count_accidents.spark_count_number_of_accidents manual__2024-10-14T08:54:13.371924+00:00 [running]> on host 14eb9bd36bc1
[2024-10-14T08:54:14.708+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='count_accidents' AIRFLOW_CTX_TASK_ID='spark_count_number_of_accidents' AIRFLOW_CTX_EXECUTION_DATE='2024-10-14T08:54:13.371924+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-14T08:54:13.371924+00:00'
[2024-10-14T08:54:14.709+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-10-14T08:54:14.709+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-10-14T08:54:14.709+0000] {logging_mixin.py:190} INFO - Current task name:spark_count_number_of_accidents state:running start_date:2024-10-14 08:54:14.609437+00:00
[2024-10-14T08:54:14.709+0000] {logging_mixin.py:190} INFO - Dag name:count_accidents and current dag run status:running
[2024-10-14T08:54:14.710+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-14T08:54:14.716+0000] {base.py:84} INFO - Retrieving connection 'spark'
[2024-10-14T08:54:14.717+0000] {spark_submit.py:401} INFO - Spark-Submit cmd: spark-submit --master spark-submit --master :7077:7077 --name arrow-spark --queue YARN --deploy-mode client ./spark/count_accidents.py
[2024-10-14T08:54:16.234+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SparkContext: Running Spark version 3.5.3
[2024-10-14T08:54:16.235+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SparkContext: OS info Linux, 6.6.32-linuxkit, aarch64
[2024-10-14T08:54:16.236+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SparkContext: Java version 17.0.12
[2024-10-14T08:54:16.283+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-10-14T08:54:16.346+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO ResourceUtils: ==============================================================
[2024-10-14T08:54:16.346+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-10-14T08:54:16.347+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO ResourceUtils: ==============================================================
[2024-10-14T08:54:16.347+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SparkContext: Submitted application: CountAccidents
[2024-10-14T08:54:16.358+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-10-14T08:54:16.362+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO ResourceProfile: Limiting resource is cpu
[2024-10-14T08:54:16.363+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-10-14T08:54:16.400+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SecurityManager: Changing view acls to: default
[2024-10-14T08:54:16.400+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SecurityManager: Changing modify acls to: default
[2024-10-14T08:54:16.401+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SecurityManager: Changing view acls groups to:
[2024-10-14T08:54:16.401+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SecurityManager: Changing modify acls groups to:
[2024-10-14T08:54:16.401+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: default; groups with view permissions: EMPTY; users with modify permissions: default; groups with modify permissions: EMPTY
[2024-10-14T08:54:16.532+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO Utils: Successfully started service 'sparkDriver' on port 38747.
[2024-10-14T08:54:16.571+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SparkEnv: Registering MapOutputTracker
[2024-10-14T08:54:16.594+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SparkEnv: Registering BlockManagerMaster
[2024-10-14T08:54:16.605+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-10-14T08:54:16.605+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-10-14T08:54:16.608+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-10-14T08:54:16.623+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-09149024-fdd5-4f72-8555-b22830a407a4
[2024-10-14T08:54:16.631+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-10-14T08:54:16.639+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-10-14T08:54:16.717+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-10-14T08:54:16.758+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-10-14T08:54:16.788+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 ERROR SparkContext: Error initializing SparkContext.
[2024-10-14T08:54:16.788+0000] {spark_submit.py:571} INFO - org.apache.spark.SparkException: Could not parse Master URL: 'spark-submit --master :7077:7077'
[2024-10-14T08:54:16.789+0000] {spark_submit.py:571} INFO - at org.apache.spark.SparkContext$.org$apache$spark$SparkContext$$createTaskScheduler(SparkContext.scala:3194)
[2024-10-14T08:54:16.789+0000] {spark_submit.py:571} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:577)
[2024-10-14T08:54:16.789+0000] {spark_submit.py:571} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2024-10-14T08:54:16.789+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2024-10-14T08:54:16.789+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2024-10-14T08:54:16.789+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-10-14T08:54:16.790+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
[2024-10-14T08:54:16.790+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
[2024-10-14T08:54:16.790+0000] {spark_submit.py:571} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2024-10-14T08:54:16.790+0000] {spark_submit.py:571} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-10-14T08:54:16.790+0000] {spark_submit.py:571} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2024-10-14T08:54:16.790+0000] {spark_submit.py:571} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2024-10-14T08:54:16.790+0000] {spark_submit.py:571} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2024-10-14T08:54:16.790+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-10-14T08:54:16.790+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-10-14T08:54:16.791+0000] {spark_submit.py:571} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-10-14T08:54:16.791+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-10-14T08:54:16.794+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SparkUI: Stopped Spark web UI at http://14eb9bd36bc1:4040
[2024-10-14T08:54:16.810+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-10-14T08:54:16.817+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO MemoryStore: MemoryStore cleared
[2024-10-14T08:54:16.817+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO BlockManager: BlockManager stopped
[2024-10-14T08:54:16.821+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-10-14T08:54:16.821+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 WARN MetricsSystem: Stopping a MetricsSystem that is not running
[2024-10-14T08:54:16.823+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-10-14T08:54:16.827+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO SparkContext: Successfully stopped SparkContext
[2024-10-14T08:54:16.828+0000] {spark_submit.py:571} INFO - Traceback (most recent call last):
[2024-10-14T08:54:16.828+0000] {spark_submit.py:571} INFO - File "/opt/airflow/spark/count_accidents.py", line 19, in <module>
[2024-10-14T08:54:16.828+0000] {spark_submit.py:571} INFO - count_accidents()
[2024-10-14T08:54:16.829+0000] {spark_submit.py:571} INFO - File "/opt/airflow/spark/count_accidents.py", line 8, in count_accidents
[2024-10-14T08:54:16.829+0000] {spark_submit.py:571} INFO - spark = SparkSession.builder.appName("CountAccidents").getOrCreate()
[2024-10-14T08:54:16.829+0000] {spark_submit.py:571} INFO - ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-10-14T08:54:16.829+0000] {spark_submit.py:571} INFO - File "/home/airflow/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 497, in getOrCreate
[2024-10-14T08:54:16.829+0000] {spark_submit.py:571} INFO - File "/home/airflow/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 515, in getOrCreate
[2024-10-14T08:54:16.830+0000] {spark_submit.py:571} INFO - File "/home/airflow/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 203, in __init__
[2024-10-14T08:54:16.830+0000] {spark_submit.py:571} INFO - File "/home/airflow/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 296, in _do_init
[2024-10-14T08:54:16.830+0000] {spark_submit.py:571} INFO - File "/home/airflow/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 421, in _initialize_context
[2024-10-14T08:54:16.830+0000] {spark_submit.py:571} INFO - File "/home/airflow/.local/lib/python3.12/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1587, in __call__
[2024-10-14T08:54:16.830+0000] {spark_submit.py:571} INFO - File "/home/airflow/.local/lib/python3.12/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2024-10-14T08:54:16.831+0000] {spark_submit.py:571} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2024-10-14T08:54:16.831+0000] {spark_submit.py:571} INFO - : org.apache.spark.SparkException: Could not parse Master URL: 'spark-submit --master :7077:7077'
[2024-10-14T08:54:16.831+0000] {spark_submit.py:571} INFO - at org.apache.spark.SparkContext$.org$apache$spark$SparkContext$$createTaskScheduler(SparkContext.scala:3194)
[2024-10-14T08:54:16.831+0000] {spark_submit.py:571} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:577)
[2024-10-14T08:54:16.831+0000] {spark_submit.py:571} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2024-10-14T08:54:16.831+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2024-10-14T08:54:16.831+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2024-10-14T08:54:16.831+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-10-14T08:54:16.831+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
[2024-10-14T08:54:16.831+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
[2024-10-14T08:54:16.832+0000] {spark_submit.py:571} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2024-10-14T08:54:16.832+0000] {spark_submit.py:571} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-10-14T08:54:16.832+0000] {spark_submit.py:571} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2024-10-14T08:54:16.832+0000] {spark_submit.py:571} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2024-10-14T08:54:16.832+0000] {spark_submit.py:571} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2024-10-14T08:54:16.832+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-10-14T08:54:16.832+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-10-14T08:54:16.832+0000] {spark_submit.py:571} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-10-14T08:54:16.833+0000] {spark_submit.py:571} INFO - 
[2024-10-14T08:54:16.856+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO ShutdownHookManager: Shutdown hook called
[2024-10-14T08:54:16.856+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO ShutdownHookManager: Deleting directory /tmp/spark-4e08fffe-1eaf-44a7-b48d-475f0440fc7d
[2024-10-14T08:54:16.858+0000] {spark_submit.py:571} INFO - 24/10/14 08:54:16 INFO ShutdownHookManager: Deleting directory /tmp/spark-33a53d24-574c-434c-b035-f4debdd8dae0
[2024-10-14T08:54:16.881+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 502, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark-submit --master :7077:7077 --name arrow-spark --queue YARN --deploy-mode client ./spark/count_accidents.py. Error code is: 1.
[2024-10-14T08:54:16.884+0000] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-10-14T08:54:16.884+0000] {logging_mixin.py:190} INFO - Task start:2024-10-14 08:54:14.609437+00:00 end:2024-10-14 08:54:16.883800+00:00 duration:2.274363
[2024-10-14T08:54:16.884+0000] {logging_mixin.py:190} INFO - Task:<Task(SparkSubmitOperator): spark_count_number_of_accidents> dag:<DAG: count_accidents> dagrun:<DagRun count_accidents @ 2024-10-14 08:54:13.371924+00:00: manual__2024-10-14T08:54:13.371924+00:00, state:running, queued_at: 2024-10-14 08:54:13.382405+00:00. externally triggered: True>
[2024-10-14T08:54:16.884+0000] {logging_mixin.py:190} INFO - Failure caused by Cannot execute: spark-submit --master spark-submit --master :7077:7077 --name arrow-spark --queue YARN --deploy-mode client ./spark/count_accidents.py. Error code is: 1.
[2024-10-14T08:54:16.884+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=count_accidents, task_id=spark_count_number_of_accidents, run_id=manual__2024-10-14T08:54:13.371924+00:00, execution_date=20241014T085413, start_date=20241014T085414, end_date=20241014T085416
[2024-10-14T08:54:16.893+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-14T08:54:16.893+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 325 for task spark_count_number_of_accidents (Cannot execute: spark-submit --master spark-submit --master :7077:7077 --name arrow-spark --queue YARN --deploy-mode client ./spark/count_accidents.py. Error code is: 1.; 1206)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 502, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark-submit --master :7077:7077 --name arrow-spark --queue YARN --deploy-mode client ./spark/count_accidents.py. Error code is: 1.
[2024-10-14T08:54:16.920+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-10-14T08:54:16.933+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-14T08:54:16.934+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
